script.js

```diff
@@ -1,74 +1,87 @@
 
-// Storage
-
-function updateStorage(data) {
-  localStorage.setItem('bookLists', JSON.stringify(data));
-}
+class Book {
+  updateStorage(data) {
+    localStorage.setItem('bookLists', JSON.stringify(data));
+  }
 
-// Remove title function
+  // Remove title function
 
-function removeBook(title) {
-  booklist = booklist.filter((book) => book.title !== title);
-  updateStorage(booklist);
-}
+  removeBook(title) {
+    const rm = new Book();
+    const index = booklist.findIndex((book) => book.title === title);
+    if (index !== -1) {
+      booklist.splice(index, 1);
+      rm.updateStorage(booklist);
+    }
+  }
 
-// Function to display the book's list
+  // Function to display the book's list
 
-function displayBooks() {
-  document.getElementById('book-list-container').innerHTML = '';
-  booklist.forEach((book) => {
-    const bookItem = document.createElement('div');
-    const booksTable = document.getElementById('book-list-container');
-    bookItem.classList.add('book-item');
-    bookItem.innerHTML = `
-        <p>Book title: ${book.title}</p>
-        <p>Author: ${book.author}</p>
-        <button class="remove" id=${book.title}>Remove</button>
-      <hr/>`;
-    booksTable.appendChild(bookItem);
+  displayBooks() {
+    document.getElementById('book-list-container').innerHTML = '';
+    booklist.forEach((book) => {
+      const bookItem = document.createElement('div');
+      const booksTable = document.getElementById('book-list-container');
+      bookItem.classList.add('book-item');
+      bookItem.innerHTML = `
+      <div>
+          <p>${book.title}, by ${book.author}</p>
+          <button class="remove" id=${book.title}>Remove</button>
+      </div>
+      <hr/>
+      `;
+      booksTable.appendChild(bookItem);
 
-    const removeBtn = bookItem.querySelector('.remove');
-    removeBtn.addEventListener('click', () => {
-      removeBook(book.title);
-      displayBooks();
+      const removeBtn = bookItem.querySelector('.remove');
+      removeBtn.addEventListener('click', () => {
+        const rm = new Book();
+        rm.removeBook(book.title);
+        rm.displayBooks();
+      });
     });
-  });
-}
+  }
 
-function loadFromStorage() {
-  const storedBookList = localStorage.getItem('bookLists');
-  if (storedBookList) {
-    booklist = JSON.parse(storedBookList);
-    displayBooks();
+  loadFromStorage() {
+    const ld = new Book();
+    const storedBookList = localStorage.getItem('bookLists');
+    if (storedBookList) {
+      booklist = JSON.parse(storedBookList);
+      ld.displayBooks();
+    }
   }
-}
 
-// Function to add new books
-function addBooks(ttle, athr) {
-  const book = { title: '', author: '' };
-  book.title = ttle;
-  book.author = athr;
-  booklist.push(book);
-  updateStorage(booklist);
+  // Function to add new books
+  addBooks(ttle, athr) {
+    const bk = new Book();
+    const book = { title: '', author: '' };
+    book.title = ttle;
+    book.author = athr;
```

Comment:
- You can generate a random number or string and add it as an `id` attribute to achieve the functionality discussed above. 🧐️